### DISCUSSION TITLE
Clickable row

### DISCUSSION THREAD

**Discussion by stefanrakicfaxi** [Power-Components/livewire-powergrid#968] [Q&A]
# ⚡ PowerGrid ⚡ Feature Request

## Summary

Very often I want to have clickable row in table 

### Why is this needed?

It's super easy for clients to use 

### Suggested Solution(s)

Add option to define route for rows

**Comment by luanfreitasdev** [comment]
Have you already taken a look here?

https://v4.livewire-powergrid.com/table/update-data.html?id=editonclick#editonclick

**Comment by paulmercs** [comment]
> # ⚡ PowerGrid ⚡ Feature Request
> ## Summary
> Very often I want to have clickable row in table
> 
> ### Why is this needed?
> It's super easy for clients to use
> 
> ### Suggested Solution(s)
> Add option to define route for rows

Have you tried this? is there any workarounds for it to work?
